packages/manager/modules/pci/src/projects/project/additional-ips/floating-ips/floating-ips.html

```diff
@@ -0,0 +1,72 @@
+<div data-ui-view>
+    <oui-datagrid data-rows="$ctrl.floatingIps">
+        <oui-datagrid-topbar>
+            <oui-button
+                variant="secondary"
+                data-on-click="$ctrl.goToAdditionalIpOrderPage()"
+            >
+                <i class="oui-icon oui-icon-plus" aria-hidden="true"></i>
+                <span
+                    data-translate="pci_additional_ips_add_additional_ip"
+                ></span>
+            </oui-button>
+        </oui-datagrid-topbar>
+        <oui-datagrid-column
+            data-title=":: 'pci_additional_ips_floating_ip_grid_ip' | translate"
+            data-property="ip"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_additional_ips_floating_ip_grid_region' | translate"
+            data-property="region"
+            data-type="string"
+            data-filterable
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_additional_ips_floating_ip_grid_associated_service' | translate"
+            data-property="associatedEntity.id"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_additional_ips_floating_ip_grid_assocated_endpoint' | translate"
+            data-property="instance.name"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+            <a
+                data-ng-if="$row.instance"
+                data-ui-sref="pci.projects.project.instances.instance({projectId : $ctrl.projectId, instanceId: $row.instance.id})"
+                data-ng-bind="$row.instance.name"
+            ></a>
+        </oui-datagrid-column>
+        <oui-action-menu text="Actions" compact>
```

Comment:
Do we not need translation for this text ?

Reply:
It is actually not required. So removed it.